### DISCUSSION TITLE
Can use javascript in QSkinny？

### DISCUSSION THREAD

**Discussion by hestrro** [uwerat/qskinny#498] [General]
QSkinny can use qml, so javascript should also be used in QSkinny. Have you tried it?

**Comment by uwerat** [comment]
It's the other way round: you can access QSkinny classes from QML - QSkinny does not need or use Qml at all.
Of course you can always use Javascript within QML - this has nothing to do with what classes offer a QML API.

However - don't expect too much from the QML API of the Qskinny classes. As the motivation behind using QSkinny is very often too avoid QML supporting it does not have highest priority and the first release won't be much more than a proof of concept.
